assets/css/v2/style.css

```diff
@@ -988,140 +990,84 @@ blockquote.side-callout {
 
 /* Codeblocks */
 .highlight {
+  padding: 0 1rem 0 1rem;
+  grid-column: 1 / -1 !important;
   position: relative;
   z-index: -1;
+  flex: 1;
 
   code .cl {
     position: relative;
-    width: 100%;
-    white-space: pre-wrap;
   }
 }
 
-.highlight-mf {
-  grid-column: 1 / -1 !important;
-  position: relative;
-  margin-left: 0;
-  z-index: -1;
-  width: calc(100% + var(--overflow-gutter-extension));
-
-  &:has(.comment) {
-    margin-left: calc(var(--overflow-gutter-extension) / -1);
-  }
-
-  code {
-    font-weight: light;
-    font-family: "JetBrains Mono", monospace;
-    display: flex;
-    flex-direction: column;
-  }
-
-  pre.chroma {
-    overflow-x: auto;
-    box-sizing: border-box;
-  }
+.code-block {
+  grid-column: 1;
+}
 
-  code .line {
-    position: relative;
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    grid-template-areas:
-      " code "
-      " comment ";
-  }
+.icon-code-copy {
+  background-color: #f2f2f2;
+  border: none;
+  padding: 5px 10px;
+  cursor: pointer;
+  font-family: "JetBrains Mono", monospace;
```

Comment:
We're not yet actually shipping `JetBrains Mono`, but it may work on systems with the font installed locally.